decidim-maintainers_toolbox/README.md

```diff
@@ -0,0 +1,52 @@
+# Decidim::MaintainersToolbox
+
+Release related tools for the Decidim project.
+
+Tools for releasing, backporting, changelog generating, and working with GitHub
+
+## Installation
+
+TODO: Replace `UPDATE_WITH_YOUR_GEM_NAME_PRIOR_TO_RELEASE_TO_RUBYGEMS_ORG` with your gem name right after releasing it to RubyGems.org. Please do not do it earlier due to security reasons. Alternatively, replace this section with instructions to install your gem from git if you do not plan to release to RubyGems.org.
+
+This gem is meant to be used outside of bundler/Gemfile so we do not need to bump the version every time we release a new one to each of the releases branch.
+
+```console
+gem install UPDATE_WITH_YOUR_GEM_NAME_PRIOR_TO_RELEASE_TO_RUBYGEMS_ORG
+```
```

Comment:
I am going to let this lines slip, as they clearly state to be changed after publication.

Reply:
Yes, this is like a security mesure from the people of bundler, so attackers can't squat the name of the gem. I mention what should the fow be in https://github.com/decidim/decidim/pull/12281#issuecomment-1893767197